apps/ledger-live-desktop/package.json

```diff
@@ -38,7 +38,9 @@
     "test:jest": "jest src --silent --ci --coverage --json --testLocationInResults --outputFile=report.json",
     "test:playwright:setup": "playwright install chromium",
     "test:playwright": "playwright test --config=tests/playwright.config.ts",
```

Comment:
Just a question: if someone runs only `pnpm test:playwright` locally, which behavior is it going to use by default ? It's going to test all playwright tests ?

Reply:
I think i tried it myself and it takes the first config described in the project key in thep laywright config file

Reply:
Ok thx for the info ! :)